src/content/docs/dropins/all/extend-or-create.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Extend, substitute, or create?
+description: Decision guidance for customizing drop-ins - when to extend existing ones, when to substitute with third-party solutions, and when creating new ones might be necessary.
+sidebar:
+  label: Extend, substitute, or create
+---
+
+import Vocabulary from '@components/Vocabulary.astro';
+import Diagram from '@components/Diagram.astro';
+import Aside from '@components/Aside.astro';
+import Callouts from '@components/Callouts.astro';
+import { Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+When customizing Commerce drop-ins, you have three main approaches. **Extending** existing drop-ins is the recommended and fully supported path. **Substituting** drop-ins with third-party solutions is sometimes necessary but comes with risks. **Creating** new drop-ins from scratch is technically possible but currently experimental and unsupported. This guide helps you choose the right approach for your specific needs.
+
+## Vocabulary
+
+<Vocabulary>
+
+### Extend
+
+Modify or enhance existing drop-ins using slots, styling, events, or configuration. This is the **recommended and fully supported** approach.
+
+### Substitute
+
+Replace an Adobe drop-in entirely with a third-party solution (like using Stripe for checkout). This comes with maintenance responsibilities and compatibility risks.
+
+### Create
+
+Build a brand-new drop-in from scratch using the SDK. Currently in **early access** - experimental with limited support.
+
+### Slots
+
+Declarative anchors where custom UI content can be injected into a drop-in.
+
+</Vocabulary>
+
+## Decision guidance
+
+Use this decision flow to determine the right approach for your needs:
+
+<Steps>
+1. **Can you achieve your goals by extending?** 
+   - Use slots to add custom content
+   - Apply custom styling and layouts
+   - Listen to events for custom behavior
+   - Modify configurations
+   
+   **If yes** → **Extend** (recommended path)
+
+2. **Do you need to completely replace core functionality?**
+   - Common with checkout flows (Stripe, PayPal, etc.)
+   - Third-party service provides their own UI/logic
```

Comment:
Even these cases have potential for usage within a slot.

Reply:
@sirugh, I don't understand. If Stripe and other third-party services replace the entire checkout process, which drop-in do you extend without also getting that drop-in's features and functions that are substituted? You can't use the Checkout drop-in because that's the drop-in you want to replace.

Reply:
The problem is with clarity of wording. There are checkout flows that do replace the entire page, and there are flows that are just "injectable" such as with payment methods.

See Paypal's docs for an example of "injectable": https://developer.paypal.com/studio/checkout/standard/integrate

So more like, if the merchant wants to replace the entire checkout flow with some all-in-one experience from a payment provider like stripe/paypal, then they would do just that. But if they want to use some of our checkout dropin/flow but integrate to Paypal/Stripe via Slots, that is also possible.

Mainly I want to make sure we don't scare people away from using our checkout just because they want to use Stripe or Paypal.